bin/trackers/Tracker_Term.py

```diff
@@ -135,8 +139,24 @@ def new_term_found(self, term, term_type, item_id, item_source):
                     print(f'S        print(item_content)end Mail {mail_subject}')
                     NotificationHelper.sendEmailNotification(mail, mail_subject, mail_body)
 
+                # Webhook
+                webhook_to_post = Term.get_term_webhook(term_uuid)
+                if webhook_to_post:
+                    json_request = {"trackerId": term_uuid,
+                                    "itemId": item_id,
+                                    "itemURL": self.full_item_url + item_id,
+                                    "term": term,
+                                    "itemSource": item_source,
+                                    "itemDate": item_date,
+                                    "tags": tags_to_add,
+                                    "emailNotification": f'{mail_to_notify}',
+                                    "trackerType": term_type
+                                    }
+                    response = requests.post(webhook_to_post, json=json_request)
+                    if response.status_code >= 400:
+                        raise Exception(f"Webhook request failed for {webhook_to_post}\nReason: {response.reason}")
```

Comment:
I don't think we should stop the module if we can't reach the webhook.
Can you replace this exception by an error message ?

You can use this function:
```python
self.redis_logger.error(error_message)
```